naga/src/front/glsl/parser/types.rs

```diff
@@ -277,14 +277,6 @@ impl ParsingContext<'_> {
                     let storage_access = qualifiers
                         .storage_access
                         .get_or_insert((crate::StorageAccess::all(), Span::default()));
-                    if !storage_access.0.contains(!access) {
-                        frontend.errors.push(Error {
-                            kind: ErrorKind::SemanticError(
-                                "The same memory qualifier can only be used once".into(),
-                            ),
-                            meta: token.meta,
-                        })
-                    }
```

Comment:
I don't know what this is for, but if I don't remove it then naga glsl snapshot fails